sklearn/svm/tests/test_svm.py

```diff
@@ -975,3 +980,22 @@ def test_ovr_decision_function():
     # Test if the first point has lower decision value on every quadrant
     # compared to the second point
     assert_true(np.all(pred_class_deci_val[:, 0] < pred_class_deci_val[:, 1]))
+
+
+def test_gamma_auto():
+    X, y = [[0.0], [1.0]], [0, 1]
+
+    msg = ("The default gamma parameter value 'auto', calculated as "
```

Comment:
I think this would be clearer as "The default value of gamma will change from 'auto' to 'scale' in version X to account better for unscaled features". The user can read the docs.

Reply:
Done.

Reply:
Actually it might be good to add "Set gamma explicitly to 'auto' or 'scale' to avoid this warning."